community/knative/charts/serving/templates/webhook-pod.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
```

Comment:
[ERROR] charts/serving/templates/queue-proxy-img.yaml: expected valid API version: alpha API version 'caching.internal.knative.dev/v1alpha1' found (NoInvalidAPIVersions)
[ERROR] charts/serving/templates/webhook-pod.yaml: expected container to have a livenessProbe: spec.template.spec.containers[0].livenessProbe not found (ContainerHasLivenessProbe)
[ERROR] charts/serving/templates/webhook-pod.yaml: expected container to have a readinessProbe: spec.template.spec.containers[0].readinessProbe not found (ContainerHasReadinessProbe)
[ERROR] charts/serving/templates/webhook-pod.yaml: missing architecture-based node affinity: expected value 'beta.kubernetes.io/arch' at some spec.template.spec.affinity.nodeAffinity.requiredDuringSchedulingIgnoredDuringExecution.nodeSelectorTerms[i].matchExpressions[j].key (PodHasArchBasedNodeAffinity)
[ERROR] charts/serving/templates/webhook-pod.yaml: missing metering annotations: missing spec.template.metadata.annotations.productID, spec.template.metadata.annotations.productName, spec.template.metadata.annotations.productVersion (MeteringAnnotationLabels)